README.md

```diff
@@ -46,7 +48,20 @@ Input: a specific point in lat/lon, and a set of depth intervals.
 
 • https://nrcs.app.box.com/v/soils/folder/17971946225
 
-### Acknowledgements
+## Testing
+
+### Regular tests
+
+There is a small suite of integration tests which can be run with the `make test` command, and gets run regularly by CI.
+
+### Bulk test
+
+There is a large suite of integration tests which takes many hours to run. It comes in the format of two scripts:
+
+-   Run `python -m soil_id.tests.us.generate_bulk_test_results` to run the algorithm over a collection of 3000 soil pits, which will accumulate the results in a log file.
+-   Run `python -m soil_id.tests.us.process_bulk_test_results $LOG_FILE_PATH` to view statistics calculated over that log file.
```

Comment:
issue: please add makefile command(s) for this.

suggestion: can this run under pytest?

Reply:
re: pytest, it doesn't really have a success or failure condition and i'm not sure how to conditionally run it (since we don't want it to run on each CI run). what would the advantage be?

Reply:
Why: It's easier to have one test mechanism than two.

How: We should be able to something like this:
```
@pytest.mark.skipif(os.environ.get("BULK_TESTS_ENABLED", "false") == "true", reason="Bulk tests not enabled")
```

suggestion: You can `assert True` at the end of the test run if there is at least one result.

Reply:
makefile is done

Reply:
re: pytest, i just don't think it fits:
- having a single 8 hour test violates my expectations of what an automated test suite is supposed to do
- usually if you're running the bulk tests you don't care about running the rest of the tests
- there's two scripts, one which just logs the results to disk so that you don't have to start over if something weird happens, and one which just processes the resulting log file. should pytest run both? just one? should there be two tests to do each step?
- you want to be able to run them separately in case you e.g. want to iterate on just the processing script but already have the log file

Reply:
the way this test works is very different from the way the tests we run under pytest work, so i think it's _less_ confusing to have two different mechanisms because the how and why of them is very different